readme.md

```diff
@@ -27,6 +27,13 @@ require __DIR__ . '/../../vendor/dg/adminer-custom/index.php'; // CHECK THAT PAT
 
 And then navigate your browser to this directory (i.e. `http://localhost/adminer`).
 
+Autologin
+---------
+
+If you want to login automatically (use that for non-public environment only, e.g. localhost), you can do that by setting environment variables `ADMINER_SERVER`, `ADMINER_USERNAME` and `ADMINER_PASSWORD`. If you want to be automatically redirected to https, set `ADMINER_HTTPS_REDIRECT` environment variable to `true`.
+
+Pro tip:
+You can set environment variables for the application with [symfony/dotenv](https://packagist.org/packages/symfony/dotenv) package (or another package that sets environment variables. Require this package with composer, autoload composer dependencies, load your `.env` file and include `index.php` as shown above. `.env` file should be put outside of `www` directory so it's not accessible via browser.
```

Comment:
Isn't it easier to call 3 times `putenv()` and include `index.php` than this procedure?

Reply:
Can be, but it's higher probability that developer will put index.php with credentials to git repository. On the other hand you can easily exclude .env file in .gitignore. Then developer have local .env file with credentials and when deploys application, it won't login automaticaly.

Reply:
I have never heard about .env files, where are they used?

Reply:
That is pretty standard when you want to solve different configuration in different environments. Every environment have env variables defined in .env file or env variables system-wise. Here are some examples how it is used:

https://laravel.com/docs/5.6/configuration#environment-configuration
https://docs.docker.com/compose/env-file/
https://devcenter.heroku.com/articles/heroku-local
https://plugins.jetbrains.com/plugin/9525--env-files-support
https://medium.com/@danieljameskay/create-react-app-dotenv-281693a19ecd
https://symfony.com/doc/current/configuration/environments.html
... etc.

The think is that configuration that differs environment by environment and/or application secrets should not be in the code, but in separate file. Ideal is to have some configuration and secrets defined as a environment variables. Those variables can be defined in file, that is not tracked by git and that can be applied by target system - e.g. loaded by docker, loaded by application itself, usualy during deployment process.

Reply:
.env file should look simply like that:

```
ADMINER_USERNAME=root
ADMINER_PASSWORD=secret
ADMINER_SERVER=localhost
```

Reply:
Okay, I'll merge it without the last sentence. It will be great if you can write an article how to use dotenv, create and load .env, etc…, but from that one sentence nobody will understand exactly what to do.